http-server/pom.xml

```diff
@@ -21,13 +21,14 @@
             <artifactId>vertx-web</artifactId>
         </dependency>
         <dependency>
-            <groupId>org.apache.logging.log4j</groupId>
-            <artifactId>log4j-api</artifactId>
+            <groupId>org.apache.kafka</groupId>
+            <artifactId>kafka-clients</artifactId>
```

Comment:
why this dependency? aren't we using the Vert.x Kafka client? Is this just for the constants? We should try to avoid this because the Kafka clients dependency is brought by the Vert.x Kafka client itself and we have the risk to import two different versions.

Reply:
Yes, it is because of constants.

Reply:
Also `new ConfigResource(org.apache.kafka.common.config.ConfigResource.Type.TOPIC, topicToDescribe);`
For some reason the `type` parameter is needed to be "kafka" and not "vertx" kind :(

Reply:
but should be this dependency brought by Vert.x? Isn't it enough having the Vert.x Kafka client dependency?